torch/_inductor/codegen/wrapper.py

```diff
@@ -1575,10 +1631,47 @@ def codegen_int_array_var(self, int_array: str, writer=None):
 
     def make_buffer_allocation(self, buffer):
         name = buffer.get_name()
-        device = self.codegen_device(buffer.get_device())
-        dtype = self.codegen_dtype(buffer.get_dtype())
-        size = self.codegen_shape_tuple(tuple(buffer.get_size()))
-        stride = self.codegen_shape_tuple(tuple(buffer.get_stride()))
+        # outputs are passed-in in the AOT mode
+        if self.use_preallocated_output(buffer):
```

Comment:
can confirm, this function doesn't seem to exist

Reply:
yeah this was an artifact of rebasing. this PR isn't intended for review yet